### ISSUE TITLE
Please write a scripts for  HF trainer and Qlora trainer

### ISSUE THREAD

**Issue by aamir-gmail** [grahamannett/finetune-fuyu#1]
Please have a script available HF trainer with either deep seed directly or accelerate .
QLORA HF trainer  with model merge and infer. Thank you .

**Comment by z3ugma** [comment]
Referencing https://huggingface.co/adept/fuyu-8b/discussions/60, would you be interested in making an Accelerate and QLoRA script @grahamannett ? 


If anyone would like to test their machine with fuyu, here is a small script that makes fake text + images but is a complete training loop. It is all self-contained and only needs transformers/torch/simple_parsing installed.
The idea is that since you may not know if the model will fit on your resources, better to try this before digging into FSDP/QLoRA/Accelerate.

I can add an FSDP/Accelerate/QLoRA example as well since those can be hard to get working with this model with limited resources.

**Comment by grahamannett** [comment]
@aamir-gmail @z3ugma there is an accelerate example using deepspeed in this repo.  what are you looking for in the qlora example as I have done that but havent looked at using the hf trainer with it

**Comment by aamir-gmail** [comment]
> @aamir-gmail @z3ugma there is an accelerated example using deepspeed in this repo. what are you looking for in the qlora example I have done that but haven't looked at using the HF trainer with it

Thank you for your reply, The reason included, HF, HF Datasets, and HF Qlora is because it tie neatly into the HF echo system,   I have written some the HF and QLORA scripts using HF Datasets, where can I send them for your review?

**Comment by grahamannett** [comment]
@aamir-gmail just PR them here. if the scripts are simple and work for me, will include them.  I've found a lot of the HF related tooling to not work as it should (if doing anything besides tutorial type implementations) or be buggy so avoid it when I can
